homeassistant/helpers/llm.py

```diff
@@ -314,6 +327,22 @@ async def async_get_api_instance(self, llm_context: LLMContext) -> APIInstance:
             custom_serializer=_selector_serializer,
         )
 
+    async def async_handle_externally(
```

Comment:
Should it be part of the LLM API?
I think it would better be a part of `intent` helper or `conversation` integration.

Reply:
Moved into pipeline